wiki/Tournaments/MCNC/2021/en.md

```diff
@@ -0,0 +1,130 @@
+| Statistician | ![][flag_CN] [Ayase Eli](https://osu.ppy.sh/users/6764156), ![][flag_CN] [\[Crz\]xz1z1z](https://osu.ppy.sh/users/10500832) |
+
+## Links
+
+- [Discussion thread (English)](https://osu.ppy.sh/community/forums/topics/1260186)
+- [Discussion thread (Chinese)](https://osu.ppy.sh/community/forums/topics/1260175)
+- [Livestream](https://live.bilibili.com/2996250)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1xgxKNKoJNEQz_zlPUf-4fyxeBDpuhfkgqxXyOxaIkpU)**
+
+## Mappools
+
+### Qualifiers
+
+**[Download the mappack here! (18 MB)](https://drive.google.com/file/d/1KYzdeiVoa11U0FYwjzqu-EKnDiA8DFr4/view?usp=sharing)**
+
+- NoMod
+  - [Petit Rabbit's - Tenkuu Cafeteria (TV Size) \[BLOOM! (No LN)\] (Hidden is fun)](https://osu.ppy.sh/b/2923231)
+  - [Makoto (CV: Komatsu Mikako) / Kasumi (CV: Minase Inori) - Mikaiketsuna omoi \[ln\] (\[Crz\]Satori)](https://osu.ppy.sh/b/2924900)
+  - [Synthwulf - Passacaglia \[Insane\] (\[Illustrious\])](https://osu.ppy.sh/b/2906275)
+  - [Applesoda - Tales of Pumpnia \[Battle\] (\[Crz\]xz1z1z)](https://osu.ppy.sh/b/2922271)
+
+## Ruleset
+
+### Tournament rules
+
+1. This is a osu!mania head-to-head individual tournament for Chinese players.
+2. This tourney will be using 4-key score v2, and will have three stages: a qualifiers stage, a double elimination group stage followed by a double elimination bracket stage.
+3. There is no limit on the number of the players that will enter. Any and all whom we deem as a "cheater" or "suspicious", screened by both the tourney and the official osu! staff, will not be allowed entry into the tournament. Note that failure to read the entire forum post during sign-up may cause the player to not be eligible until the issue is resolved, so please read the entire post first!
+4. This tournament is for Chinese players only, including players from Mainland Chinese territories, Taiwan territories, Hong Kong territories, Macau territories. Please note that this tournament has nothing to do with politics and views about the territories of China. Only players with the flags of the territories listed above is eligible for playing.
+5. The discord server will be the source of pretty much all the information for this tournament, so if you're joining, either as player, staff, or even as a spectator, you'll want to join for all of the latest updates. Players are required to join the official discord to be eligible of playing in the tournament. For those who cannot access discord, we have a QQ server containing the methods which could help you better access discord. QQ server number: 597317576.
+6. For all of the details of the rules of the tourney, refer to the Information document here or in the discord channel. The discord channel and that document will have all the information you need, and the discord channel will be where future announcements will be made! So make sure you join the channel if you are participating.
```

Comment:
all instances of Discord should be capitalized

(`discord` --> `Discord`)

Reply:
Done!